processing/src/main/java/org/apache/carbondata/processing/sort/sortdata/SortIntermediateFileMerger.java

```diff
@@ -101,6 +87,10 @@ private void startIntermediateMerging(File[] intermediateFiles) {
         + '_' + parameters.getRangeId() + '_' + System.nanoTime()
         + CarbonCommonConstants.MERGERD_EXTENSION);
     IntermediateFileMerger merger = new IntermediateFileMerger(parameters, intermediateFiles, file);
+    if (LOGGER.isDebugEnabled()) {
+      LOGGER.debug("Sumitting request for intermediate merging no of files: "
```

Comment:
*submitting
*change "no" to number of files

Reply:
ok